### DISCUSSION TITLE
Importing UnoCSS from CSS

### DISCUSSION THREAD

**Discussion by silverwind** [unocss/unocss#3042] [Q&A]
I'd like to import UnoCSS from a CSS file with `@import` in Vite. In my `index.css`:

```css
@import "virtual:uno.css";
```

This fails to build with both postcss and lightningcss preprocessors with this error:

```
ENOENT: no such file or directory, open 'virtual:uno.css'
```

Is this or can this be supported? I prefer to import CSS only from CSS, one being [this](https://github.com/vitest-dev/vitest/issues/3862).

**Comment by Shyam-Chen** [comment]
https://vitejs.dev/guide/api-plugin.html#virtual-modules-convention

**Comment by silverwind** [comment]
Not sure what that link is supposed to tell me, but I got it to work with `@unocss/postcss`, though with warnings regarding a missing entry module, not sure how to get rid of that.

**Comment by silverwind** [comment]
Actually got rid of the warning by removing the vite plugin.
